packages/cli/src/commands/org/config/pull.ts

```diff
@@ -45,11 +47,17 @@ export default class Pull extends Command {
       description:
         'The unique identifier of the snapshot to pull. If not specified, a new snapshot will be created.',
     }),
+    git: flags.boolean({
+      char: 'g',
+      description:
+        'Wether to create a git repository when creating a new project.',
```

Comment:
I do not think you meant ['castrated ram'](https://www.google.com/search?q=wether&oq=wether&aqs=chrome..69i57j0i271l3.252j0j7&sourceid=chrome&ie=UTF-8) 😂 

```suggestion
        'Whether to create a git repository when creating a new project.',
```